### PR TITLE
Add basics aa1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the repository is our template.

**What is the new behavior (if this is a feature change)?**
- It is now possible to login.
- It is now possible to add human ressources and pass on parameters regarding their basic trust values, skills and data.
- It is now possible to sort human ressources based on their skills.

**Other information**:
- The version has been set to `v1.0.0`.

### CODE REVIEW

**Review by simonprast — APPROVED** [review]
LGTM

File: src/App.js [hunk 1]
```diff
@@ -45,12 +45,10 @@ class App extends React.Component {
       <Router>
         <ScrollToTop>
           <div className="flyout">
-            <Navbar />
```
Comment (kleberbaum) [inline src/App.js:48]: Could be removed from the imports, cause never used.

File: src/App.js [hunk 2]
```diff
@@ -45,12 +45,10 @@ class App extends React.Component {
...
             <main>
               <Routes />
               <CookieModal saveCookie={this.saveCookie} />
             </main>
-            <Footer />
```
Comment (kleberbaum) [inline src/App.js:53]: Could be removed from the imports, cause never used.

File: src/Routes.js [hunk 3]
```diff
@@ -22,8 +22,7 @@ class Routes extends React.Component {
   render() {
     return (
       <Switch>
-        <Route exact path="/" component={HomePage} />
```
Comment (kleberbaum) [inline src/Routes.js:25]: Could be removed from the imports, cause never used.

File: src/components/molecules/CreateContact/index.jsx [hunk 4]
```diff
@@ -0,0 +1,577 @@
...
+                </div>
+              </form>
+            </>
+          )}
+        </MDBModalBody>
+      </MDBModal>
+    );
+  }
+}
+//#endregion
```
Comment (kleberbaum) [inline src/components/molecules/CreateContact/index.jsx:556]: Please add a newline?

File: src/components/molecules/CreateContact/index.jsx [hunk 5]
```diff
@@ -0,0 +1,577 @@
...
+//#region > Redux Mapping
+const mapDispatchToProps = (dispatch) => {
+  return {
+    createContact: (contact) => dispatch(createContact(contact)),
+  };
+};
+//#endregion
```
Comment (kleberbaum) [inline src/components/molecules/CreateContact/index.jsx:563]: Please add a newline?

File: src/components/pages/ProfilePage/index.jsx [hunk 6]
```diff
@@ -24,9 +35,216 @@ import { MDBContainer, MDBBtn } from "mdbreact";
 // To be added here
 //#endregion
 
+const AVAILABLE_SKILLS = [
```
Comment (kleberbaum) [inline src/components/pages/ProfilePage/index.jsx:39]:
```suggestion
//#region > Config
const AVAILABLE_SKILLS = [
```

File: src/components/pages/ProfilePage/index.jsx [hunk 7]
```diff
@@ -24,9 +35,216 @@ import { MDBContainer, MDBBtn } from "mdbreact";
...
+    label: "Web frontend",
+    value: "webfrontend",
+    type: "web",
+  },
+  {
+    label: "Wordpress",
+    value: "webwordpress",
+    type: "web",
+  },
+];
```
Comment (kleberbaum) [inline src/components/pages/ProfilePage/index.jsx:120]:
```suggestion
];
//#endregion
```

File: src/components/pages/ProfilePage/index.jsx [hunk 8]
```diff
@@ -24,9 +35,216 @@ import { MDBContainer, MDBBtn } from "mdbreact";
...
+
+    this.setState({
+      users: res,
+      allUsers: res,
+    });
+  };
+
+  calculateTrust = (trust) => {
+    console.log(trust);
+
```
Comment (kleberbaum) [inline src/components/pages/ProfilePage/index.jsx]:
```suggestion
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**